### PR TITLE
ENH: Add week_3_quiz.md

### PR DESCRIPTION
**Pull request by richford**
This PR adds the Week 3 quiz questions. Tagging @jdkent as a reviewer to confirm that I got the ABCD questions right.

### CODE REVIEW

File: week_3/week_3_quiz.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+First, we will ask you a few questions about ABCD Data.
+
+**Question 1**
+
+The ABCD sample is representative of youth in the United States.
+
+- True
+- False
+
+**Quesiton 2**
```
Comment (tsalo) [inline week_3/week_3_quiz.md:20]:
```suggestion
**Question 2**
```

File: week_3/week_3_quiz.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+RUN wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /opt/miniconda.sh
+RUN /bin/bash /opt/miniconda.sh -b -p /opt/conda
+ENV PATH="/opt/conda/bin:$PATH"
+```
+
+Build this docker image by navigating to the same directory as your Dockerfile and running `docker build -t my_custom_fsl .` Don't forget about the dot at the end, which tells Docker where to find the Dockerfile. Next register for an account on Docker Hub. Log into Docker Hub in your console using `docker login`. Build your image with the name `<username>/my_custom_fsl` and push your image to Docker Hub using `docker push <username>/my_custom_fsl.` Find a link to your image on Docker Hub and paste below
+
+---
+<input type="text" id="name" name="name"/>
+---
```
Comment (tsalo) [inline week_3/week_3_quiz.md]: I see the break in the rendered file but this doesn't seem to render the way you want.

**Review by tsalo — CHANGES_REQUESTED** [review]
Couple of small things.

File: week_3/week_3_quiz.md (hunk 2 above)
Comment (richford) [reply week_3/week_3_quiz.md]: Thanks. I fixed this in the latest commit and chose to get rid of the text box entirely.

**Comment by richford** [comment]
Also, FWIW, see the last extra credit question for an example of content folding. If/when we decide to come back and add the answers, we can do it this way so that we don't spoil the questions for new viewers.

File: week_3/week_3_quiz.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+Some of these questions are adapted, with modification, from the [Software Carpentries](http://software-carpentry.org) lesson on [Reproducible Computational Environments using Containers](https://carpentries-incubator.github.io/docker-introduction/index.html), which is licensed under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/legalcode). Other questions were adapted, with modification, from the [Neurohackweek](http://neurohackweek.github.io/) lesson on [Docker for Scientists](https://neurohackweek.github.io/docker-for-scientists/), which is licensed under the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/legalcode).
+
+The following questions require Docker to be installed on your computer. Please [download Docker](https://www.docker.com/products/docker-desktop) and then follow the installation instructions.
+
+**Question 6**
+
+Once Docker is installed, try downloading and running the alpine Docker container. Which of the following terminal commands would download and run the alpine Docker container? (Select all that apply)
+
+- [ ]  `docker run alpine`
```
Comment (jdkent) [inline week_3/week_3_quiz.md:66]: Do you want the checkboxes for all the questions? I'm fine with or without. Or does this specifically represent the possibility you can select multiple answers? (in which case a question in [quiz_2 should be changed](https://github.com/ABCD-ReproNim/exercises/blame/6b7f012ae33b006346b3230f75cc28b397edfd84/week_2/week_2_quiz.md#L21), but not the responsibility of this pull request I think)

Comment (dmoracze) [reply week_3/week_3_quiz.md:66]: Ahh, I spend like 10 seconds looking for how to do the square bullet but didn't find it. I like it and I can change week_2 to match

Comment (richford) [reply week_3/week_3_quiz.md]: I changed the bullet points to radio buttons using html and kept the checkboxes for ones where multiple answers can be selected at once.

**Comment by jdkent** [comment]
I cannot see the radio button on my browser, does it render for others?
image

**Comment by richford** [comment]
ARGHHH! You're right, it doesn't render for me either. It worked locally for me but not on GitHub. I'll revert.

**Comment by jdkent** [comment]
The lack of features in github markdown has bitten me several times in the past too.

**Review by jdkent — APPROVED** [review]
LGTM!

**Review by dmoracze — APPROVED** [review]
Looks good!

**Review by tsalo — APPROVED** [review]

**Pull request merged**